lib/beaker/hypervisor/docker.rb

```diff
@@ -102,7 +113,7 @@ def provision
         fix_ssh(container) if @options[:provision] == false
 
         @logger.debug("Starting container #{container.id}")
-        container.start({"PublishAllPorts" => true, "Privileged" => true})
```

Comment:
`container_opts` is the parameter used in `::Docker::Container.create(container_opts)` on [line 104](https://github.com/rishijavia/beaker-docker/blob/047407f64220862e8c7be9c2e728ee2b77c39e90/lib/beaker/hypervisor/docker.rb#L104) of this PR, so it is definitely still being used. I'm just wondering if the options parameter provided there are read in the same way as they are when provide to `start`.